### PR TITLE
Vtep6

### PR DESCRIPTION
**Pull request by outscale-mgo**

### CODE REVIEW

File: src/utils/ip.h [hunk 1]
```diff
@@ -50,9 +50,50 @@ union pg_ipv6_vtc {
 		_Generic((ip), PG_IP_GENERIC_IPV6(ip), uint32_t : &(ip)) \
 		))
 
+#define PG_UDP_PROTOCOL_NUMBER 17
+#define PG_TCP_PROTOCOL_NUMBER 6
+
 #define PG_IPV6_HALF_FORMAT "%02x%02x:%02x%02x:%02x%02x:%02x%02x"
 #define PG_IPV6_FORMAT PG_IPV6_HALF_FORMAT":"PG_IPV6_HALF_FORMAT
```
Comment (outscale-toa) [inline src/utils/ip.h]: Can you comment this function please?

File: src/utils/ip.h [hunk 2]
```diff
@@ -50,9 +50,50 @@ union pg_ipv6_vtc {
...
+	ip_hdr.src_addr = src_ip;
+	ip_hdr.dst_addr = 0;
+	return __rte_raw_cksum(&ip_hdr, sizeof(ip_hdr), 0);
+}
+
+/**
+ * Assume that all udp ip packets are similar with only
+ * packet_len, packet_id and dst_addr different from other packets
+ * from this assumption, do a fast checksum
+ */
```
Comment (outscale-toa) [inline src/utils/ip.h]: Can you change comment like : 
 /**
  *  @param	packet_len  (description)
  *  ........
*/

File: src/vtep-internal.c [hunk 3]
```diff
@@ -131,40 +130,47 @@ struct vtep_state {
  * @param	dst_ip the destination IP
  * @param	datagram_len the lenght of the datagram including the header
  */
```
Comment (outscale-toa) [inline src/vtep-internal.c]: Missing  "state" parametter in the comment

File: src/utils/ip.h (hunk 1 above)
Comment (outscale-toa) [inline src/utils/ip.h]: Can you comment this function please?

File: src/vtep-internal.c (hunk 3 above)
Comment (outscale-mgo) [reply src/vtep-internal.c]: and a pretty much unrelated change anyway
I'm just going to remove this change.

**Pull request closed without merging**